common/api/imodeljs-backend.api.md

```diff
@@ -1258,10 +1256,10 @@ export interface DownloadJob {
 
 // @internal
 export interface DownloadRequest {
-    aliasFiles?: string[];
-    checkpoint: CheckpointProps;
+    readonly aliasFiles?: string[];

```

Comment:
Should `aliasFiles` be a `ReadonlyArray<string>`?

Reply:
probably. `readonly aliasFiles?: ReadonlyArray<string>`

Reply:
done